prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -390,6 +414,34 @@ class FHIRRouter(
         return if (reverseFilter) !result else result
```

Comment:
OMG! I just noticed this. This propery is `reverseTheQualityFilter` which means it only applies to the quality filter.  You can open a new ticket for this if you want.

Reply:
Ticket written!